Gruntfile.js

```diff
@@ -0,0 +1,352 @@
+module.exports = function(grunt) {
+
+    var fs = require('fs');
+
+    var awsConfig = {
+        accessKey: false,
+        secretKey: false,
+        region: false,
+        s3Bucket: false
+    };
+
+    // Breaks out the keys and also gives us a nice reference of what
+    // needs to be in here without being able to be changed.
+    var awsConfigKeys = Object.keys(awsConfig);
+
+    // Take inputs for the awsConfig, prefixed with 'aws.'.
+    // For example accessKey can be inputted with '--aws.accessKey=accessKey'
```

Comment:
Is this how codeship prefers we pass in the credentials? I know we've also used a credentials file in the past.